src/stelnet/board/commodity/table/profit/ProfitCalculator.java

```diff
@@ -0,0 +1,37 @@
+package stelnet.board.commodity.table.profit;
+
+import com.fs.starfarer.api.campaign.econ.CommodityOnMarketAPI;
+import com.fs.starfarer.api.campaign.econ.MarketAPI;
+import stelnet.board.commodity.price.Price;
+import stelnet.board.commodity.price.SupplyPrice;
+import stelnet.util.TableCellHelper;
+
+public class ProfitCalculator {
```

Comment:
Perhaps `ProfitHelper`?

Reply:
Having "Helper" or "Util" in your class name abstracts away the actual intent and functionality of the class. What ends up happening is that, you will have Helper classes that start violating Single Responsibility Principle. If the name would describe the precise goal of a class then your inner programmer-senses would guard you against violating that principle. If the name is generic then the classes become generic, which in turn leads to badly managed code.